ani-cli

```diff
@@ -18,8 +18,7 @@
 # Project repository: https://github.com/pystardust/ani-cli
 
 # Version number
-VERSION="2.0.2"
-
+VERSION="2.0.3"
```

Comment:
2.0.4, since new keys will definitely be merged first

Reply:
Oky.. on it